### PR TITLE
Inclusion of the translation into Portuguese of Brazil.

### PR DESCRIPTION
**Pull request by keviocastro**

### CODE REVIEW

File: i18n/pt-br.xliff [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  <file original="global" source-language="en" target-language="pt-br" datatype="plaintext">
+    <header />
+    <body>
+        <trans-unit id="i-am-homepage">
+            <source><![CDATA[/^(?:|I )am on (?:|the )homepage$/]]></source>
+            <target><![CDATA[/^(?:|Eu )estou na página inicial$/]]></target>
+        </trans-unit>
+        <trans-unit id="i-go-to-homepage">
+            <source><![CDATA[/^(?:|I )go to (?:|the )homepage$/]]></source>
+            <target><![CDATA[/^(?:|Eu )vou a página inicial$/]]></target>
```
Comment (fonsecas72) [inline i18n/pt-br.xliff]: it should be ```vou à página inicial```

File: i18n/pt-br.xliff [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+            <source><![CDATA[/^(?:|I )should not see an? "(?P<element>[^"]*)" element$/]]></source>
+            <target><![CDATA[/^(?:|Eu )não devo ver o elemento "(?P<element>[^"]*)"$/]]></target>
+        </trans-unit>
+        <trans-unit id="i-should-see-num-elements">
+            <source><![CDATA[/^(?:|I )should see (?P<num>\d+) "(?P<element>[^"]*)" elements?$/]]></source>
+            <target><![CDATA[/^(?:|Eu )devo ver (?P<num>\d+) elementos? "(?P<element>[^"]*)"$/]]></target>
+        </trans-unit>
+        <trans-unit id="the-response-status-code-should-be">
+            <source><![CDATA[/^the response status code should be (?P<code>\d+)$/]]></source>
+            <target><![CDATA[/^o codigo de status da resposta deve ser (?P<code>\d+)$/]]></target>
```
Comment (fonsecas72) [inline i18n/pt-br.xliff]: It should be ```código```.
@keviocastro couldn't we simplify it to ```o código da resposta deve ser```` ?

**Comment by stof** [comment]
I will let you choose the best translation together, given that I don't speak Portuguese. Tell me when you're done.

**Comment by keviocastro** [comment]
@fonsecas72 
I agree with you.
If you have any comments, they are welcome.

File: i18n/pt-br.xliff [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+            <source><![CDATA[/^(?:|I )move backward one page$/]]></source>
+            <target><![CDATA[/^(?:|Eu )volto uma página$/]]></target>
+        </trans-unit>
+        <trans-unit id="move-forward-one-page">
+            <source><![CDATA[/^(?:|I )move forward one page$/]]></source>
+            <target><![CDATA[/^(?:|Eu )avanço uma página$/]]></target>
+        </trans-unit>
+        <trans-unit id="i-press-button">
+            <source><![CDATA[/^(?:|I )press "(?P<button>(?:[^"]|\\")*)"$/]]></source>
+            <target><![CDATA[/^(?:|Eu )pressiono o botão"(?P<button>(?:[^"]|\\")*)"$/]]></target>
```
Comment (fonsecas72) [inline i18n/pt-br.xliff]: I think there is a missing space between ```botão``` and the ```"(?P<b...``` expression.

File: i18n/pt-br.xliff [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+        </trans-unit>
+        <trans-unit id="i-follow-link">
+            <source><![CDATA[/^(?:|I )follow "(?P<link>(?:[^"]|\\")*)"$/]]></source>
+            <target><![CDATA[/^(?:|Eu )sigo o link "(?P<link>(?:[^"]|\\")*)"$/]]></target>
+        </trans-unit>
+        <trans-unit id="i-fill-in-field-with-value">
+            <source><![CDATA[/^(?:|I )fill in "(?P<field>(?:[^"]|\\")*)" with "(?P<value>(?:[^"]|\\")*)"$/]]></source>
+            <target><![CDATA[/^(?:|Eu )preencho o campo"(?P<field>(?:[^"]|\\")*)" com "(?P<value>(?:[^"]|\\")*)"$/]]></target>
```
Comment (fonsecas72) [inline i18n/pt-br.xliff]: Missing space too. ```campo "```

**Comment by fonsecas72** [comment]
@keviocastro good job, just fix the missing spaces and missing accents.

**Comment by keviocastro** [comment]
@stof believe is completed.
@fonsecas72 you agree?

File: i18n/pt-br.xliff [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+            <source><![CDATA[/^the response should not contain "(?P<text>(?:[^"]|\\")*)"$/]]></source>
+            <target><![CDATA[/^a resposta não deve conter "(?P<text>(?:[^"]|\\")*)"$/]]></target>
+        </trans-unit>
+        <trans-unit id="the-field-should-contain-value">
+            <source><![CDATA[/^the "(?P<field>(?:[^"]|\\")*)" field should contain "(?P<value>(?:[^"]|\\")*)"$/]]></source>
+            <target><![CDATA[/^o campo "(?P<field>(?:[^"]|\\")*)" deve conter o valor "(?P<value>(?:[^"]|\\")*)"$/]]></target>
+        </trans-unit>
+        <trans-unit id="the-field-should-not-contain-value">
+            <source><![CDATA[/^the "(?P<field>(?:[^"]|\\")*)" field should not contain "(?P<value>(?:[^"]|\\")*)"$/]]></source>
+            <target><![CDATA[/^o campo "(?P<field>(?:[^"]|\\")*)" não deve conter o valor"(?P<value>(?:[^"]|\\")*)"$/]]></target>
```
Comment (fonsecas72) [inline i18n/pt-br.xliff]: missing space after "o valor"

Comment (keviocastro) [inline i18n/pt-br.xliff]: Fixed.

2015-03-09 14:32 GMT-03:00 Hugo Fonseca <notifications@github.com>:

> In i18n/pt-br.xliff
> <https://github.com/Behat/MinkExtension/pull/198#discussion_r26056805>:
>
> > +        </trans-unit>
> > +        <trans-unit id="i-should-not-see-text">
> > +            <source><![CDATA[/^(?:|I )should not see "(?P<text>(?:[^"]|\\")*)"$/]]></source>
> > +            <target><![CDATA[/^(?:|Eu )não devo ver o texto "(?P<text>(?:[^"]|\\")*)"$/]]></target>
> > +        </trans-unit>
> > +        <trans-unit id="the-response-should-not-contain">
> > +            <source><![CDATA[/^the response should not contain "(?P<text>(?:[^"]|\\")*)"$/]]></source>
> > +            <target><![CDATA[/^a resposta não deve conter "(?P<text>(?:[^"]|\\")*)"$/]]></target>
> > +        </trans-unit>
> > +        <trans-unit id="the-field-should-contain-value">
> > +            <source><![CDATA[/^the "(?P<field>(?:[^"]|\\")*)" field should contain "(?P<value>(?:[^"]|\\")*)"$/]]></source>
> > +            <target><![CDATA[/^o campo "(?P<field>(?:[^"]|\\")*)" deve conter o valor "(?P<value>(?:[^"]|\\")*)"$/]]></target>
> > +        </trans-unit>
> > +        <trans-unit id="the-field-should-not-contain-value">
> > +            <source><![CDATA[/^the "(?P<field>(?:[^"]|\\")*)" field should not contain "(?P<value>(?:[^"]|\\")*)"$/]]></source>
> > +            <target><![CDATA[/^o campo "(?P<field>(?:[^"]|\\")*)" não deve conter o valor"(?P<value>(?:[^"]|\\")*)"$/]]></target>
>
> missing space after "o valor"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Behat/MinkExtension/pull/198/files#r26056805>.
>



-- 
Kévio Castro
skype: kevio.castro
(62) 8419-2439

**Comment by fonsecas72** [comment]
:+1: @keviocastro

**Comment by keviocastro** [comment]
@stof the translation is complete.

**Comment by stof** [comment]
Thanks @keviocastro and @fonsecas72.

**Comment by stof** [comment]
note that I squashed commits when merging to keep the history clean, but the authorship is preserved: https://github.com/Behat/MinkExtension/commit/ac9e16167eadd687476ea8eec0fbd94a4d1321cb

**Pull request closed without merging**